guides/getting-started/installation/react-native/ios.md

```diff
@@ -38,3 +38,13 @@ framework.
 ```console
 pod install --project-directory ios
 ```
+
+Additionally, you need to ensure the following:
+
+- `ENABLE_BITCODE` is set to false
+- `Indy.Framework` is added to the project as embedded content
+
+You can set both in XCode.app by navigating to your project settings. `ENABLE_BITCODE` should be disabled by default.
+Under the `General` tab in settings you can find `Frameworks, Libraries, and Embedded Content`. Click `+` to open the file selector and add `Indy.Framework` from your Pods.
```

Comment:
We're duplicating the installation instructions from the indy-sdk-react-native repo here. Wouldn't it be easier if we linked to that repo? 

The `ENABLE_BITCODE` piece isn't in that repo yet, and I think it would be useful to have it there.

Reply:
Can do. The  indy-sdk-react-native docs this duplicates is from a PR that was merged this morning from the same context. (I also opened that)

Reply:
Personally don't really have a strong opinion whether to include this here or rather link. Would tend to it being ok to be duplicate so people reading the docs don't need to go to another place to look at the instructions. AT the same time, when things change it only changes in one place instead of two 🤷‍♂️. What do you think?